result/yoda.js

```diff
@@ -0,0 +1,27 @@
+if ("red" === color) {
```

Comment:
__[development][eslint]__
* Error - Expected literal to be on the right side of ===. (yoda)
* Error - 'color' is not defined. (no-undef)